specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNode.json

```diff
@@ -0,0 +1,957 @@
+        }
+      ]
+    },
+    "DscNodeProperties" : {
+      "description": "The properties of a DscNode",
+      "properties": {
+        "lastSeen": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the last seen time of the node."
+        },
+        "registrationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the registration time of the node."
+        },
+        "ip": {
+          "type": "string",
+          "description": "Gets or sets the ip of the node."
+        },
+        "accountId": {
+          "type": "string",
+          "description": "Gets or sets the account id of the node."
+        },
+        "nodeConfiguration": {
+          "$ref": "#/definitions/DscNodeConfigurationAssociationProperty",
+          "description": "Gets or sets the configuration of the node."
+        },
+        "status": {
+          "type": "string",
+          "description": "Gets or sets the status of the node."
+        },
+        "nodeId": {
+          "type": "string",
+          "description": "Gets or sets the node id."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets or sets the etag of the resource."
+        },
+        "extensionHandler": {
+          "type": "array",
+          "description": "Gets or sets the list of extensionHandler properties for a Node.",
+          "items": {
+            "$ref": "#/definitions/DscNodeExtensionHandlerAssociationProperty"
+          }
+        }
+      }
+    },
+    "DscNodeExtensionHandlerAssociationProperty": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the extension handler."
+        },
+        "version": {
+          "type": "string",
+          "description": "Gets or sets the version of the extension handler."
+        }
+      },
+      "description": "The dsc extensionHandler property associated with the node"
+    },
+    "DscNodeConfigurationAssociationProperty": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the dsc nodeconfiguration."
+        }
+      },
+      "description": "The dsc nodeconfiguration property associated with the entity."
+    },
+    "AgentRegistrationRegenerateKeyParameter": {
+      "properties": {
+        "keyName": {
+          "type": "string",
+          "description": "Gets or sets the agent registration key name - primary or secondary.",
+          "enum": [
+            "primary",
+            "secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AgentRegistrationKeyName",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the resource."
+        },
+        "location": {
```

Comment:
>location [](start = 9, length = 8)

So, when regenerating keys, could the location also be updated?

Reply:
Checking with the team.

Reply:
Fixed.